app/js/components/collection_widget.ts

```diff
@@ -125,6 +126,20 @@ class CollectionWidget {
     $addEntryButton.on('click', handleAddClick);
   }
 
+  /**
+   * Add click handler to add entry
+   * @param $addEntryButton
+   */
+  private registerKeyDownEnterHandler($input: JQuery<HTMLElement>) {
+    const handleKeydownEnter = (event: JQuery.Event) => {
+      if (event.keyCode === 13) {
```

Comment:
Use of 'keyCode' property is deprecated. Please update, and use the 'which' prop.

Reply:
Reading into this, the which property is also deprecated from the [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which) standard. Might be updated soon in Jquery too :confused:

Reply:
Used the `key` property, hopefully that property will last a bit longer.